src/tide-tables/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import Header from './header'
+import TideRow from './tide-row'
+import ModalHeader from '../common/modal-header'
+
+class Location extends Component {
+  state = {
+    tides: JSON.parse(localStorage.getItem('tides')),
+  }
+
+  render() {
+    if (!this.state.tides) {
```

Comment:
I don't think I need this right? this will only show if the entry screen loads, else you'll never get to this.